spec/language/Ch04_Testbench_Templates.md

```
@@ -0,0 +1,807 @@
+    }
+  }
+}
+```
+
+**Exit Codes:**
+- `0`: All constraints satisfied
+- Non-zero: One or more constraints failed
+
+**Output Format:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)